eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -1733,7 +1739,7 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
   /**
    * This function is used to return feedback to user at channel opening
    */
-  def replyToUser(message: Either[Channel.ChannelError, String]): Unit = {
+  def replyToUser(message: Either[ChannelOpenError, ChannelCommandResponse]): Unit = {
```

Comment:
Maybe this method should be renamed to make it clear this is used at the channel opening.